site/docs/09-math/07-vector.mdx

```diff
@@ -62,168 +62,21 @@ console.log(anotherPoint.toString()) // "(50, 50)"
 
-```typescript
-const rightVector = Vector.fromAngle(0); // (1, 0)
-const upVector = Vector.fromAngle(-Math.PI / 2); // (0, -1)
-```
-
-#### `Vector.distance(vec1: Vector, vec2: Vector): number`
-Calculates the distance between two vectors.
-
-```typescript
-const distance = Vector.distance(vec(0, 0), vec(3, 4)); // 5
-```
-
-#### `Vector.isValid(vec: Vector): boolean`
-Checks if a vector is valid (not null, undefined, or containing NaN/Infinity).
-
-```typescript
-const isValid = Vector.isValid(new Vector(1, 2)); // true
-const isInvalid = Vector.isValid(new Vector(NaN, 1)); // false
-```
-
-#### `Vector.min(vec1: Vector, vec2: Vector): Vector`
-Returns a vector with the minimum x and y components.
-
-#### `Vector.max(vec1: Vector, vec2: Vector): Vector`
-Returns a vector with the maximum x and y components.
-
-### Instance Properties
-
-#### `x: number` and `y: number`
-Get or set the vector components. **Warning**: Setting these properties mutates the vector.
-
-#### `magnitude: number`
-Get or set the vector's magnitude (length). Setting this normalizes the vector and scales it to the new magnitude.
-
-### Core Operations
-
-#### `add(v: Vector, dest?: Vector): Vector`
-Adds another vector to this one.
-
-```typescript
-const result = vec(1, 2).add(vec(3, 4)); // (4, 6)
-```
-
-#### `sub(v: Vector, dest?: Vector): Vector`
-Subtracts another vector from this one.
-
-```typescript
-const result = vec(5, 3).sub(vec(2, 1)); // (3, 2)
-```
-
-#### `scale(size: number | Vector, dest?: Vector): Vector`
-Scales the vector by a scalar or component-wise by another vector.
-
-```typescript
-const scaled = vec(2, 3).scale(2); // (4, 6)
-const componentScaled = vec(2, 3).scale(vec(2, 0.5)); // (4, 1.5)
-```
-
-#### `normalize(): Vector`
-Returns a unit vector in the same direction.
-
-```typescript
-const unit = vec(3, 4).normalize(); // (0.6, 0.8)
-```
-
-### Distance and Magnitude
-
-#### `distance(v?: Vector): number`
-Returns the distance to another vector, or magnitude if no vector provided.
-
-```typescript
-const dist = vec(0, 0).distance(vec(3, 4)); // 5
-const magnitude = vec(3, 4).distance(); // 5
-```
+The standard formula for linear interpolation is:
 
-#### `squareDistance(v?: Vector): number`
-Returns the squared distance (faster than distance for comparisons).
-
-### Vector Math
-
-#### `dot(v: Vector): number`
-Calculates the dot product with another vector.
-
-```typescript
-const dotProduct = vec(1, 2).dot(vec(3, 4)); // 11
-```
-
-#### `cross(v: Vector | number): number | Vector`
-Calculates the cross product. With a vector, returns a scalar. With a scalar, returns a vector.
-
-```typescript
-const crossScalar = vec(1, 2).cross(vec(3, 4)); // -2
-const crossVector = vec(1, 2).cross(5); // (10, -5)
+```sh twoslash
```

Comment:
If the snippet isn't meant to be copied or referenced (or isn't TypeScript), you don't need to make it a twoslash directive. 
```suggestion
```sh
```